### ISSUE TITLE
Logo in recommended png format and size does not show

### ISSUE THREAD

**Issue by AlexShapka** [authentic-theme/authentic-theme#1203]
Hi Ilia, 

I've prepared a png logo of recommended size (180x90), however it is not showing up on the bottom part of left sidebar as it usually would do. I also tried to reduce image sizes slightly, but it didn't help. Is the feature broken or something wrong with my image?

**Comment by AlexShapka** [comment]
Just in case, if you want to check what's wrong with the file:

![altagrade](https://user-images.githubusercontent.com/9372724/46802713-c62d0e80-cd66-11e8-8276-2523f717b774.png)

**Comment by rostovtsev** [comment]
![screenshot from 2018-10-12 09-31-27](https://user-images.githubusercontent.com/4426533/46852042-405ea100-ce02-11e8-8e1b-46d10dd9ba9b.png)

**Comment by AlexShapka** [comment]
Webmin version | 1.890 | Usermin version | 1.741
Virtualmin version | 6.04 | Theme version | Authentic Theme 19.19

**Comment by AlexShapka** [comment]
Attempting to upgrade to 19.20 has given:

<div class="messenger-message-inner"><i class="fa fa-lg fa-fw fa-exclamation-triangle"></i>Installing this version of Authentic Theme <em>19.20-beta5</em> requires latest and/or possibly unreleased version of Host Manager <em>1.891</em> to work properly. There might be incompatible changes, that could stop the theme from working as designed. To bypass this warning, and force install the theme anyway <a data-git="1" data-stable="0" data-force="1" class="authentic_update text-darker" href="javascript:;">click here</a>, or upgrade Host Manager to the latest development  version first using console, by running <kbd>update-from-repo.sh</kbd> script from its directory.</div>

**Comment by AlexShapka** [comment]
Just to test I decided to upgrade (which didn't go through well as it broke the UI, but it's ok as it's just a testing server) What is strange that the logo showed for just an instance and then vanished again. So probably it is there, but normally you can not see.
<img width="619" alt="screen shot 2018-10-12 at 8 14 44 pm" src="https://user-images.githubusercontent.com/9372724/46883804-cf4bd780-ce5b-11e8-8572-795f41534175.png">

**Comment by rostovtsev** [comment]
You are right. Unfortunately version 19.19 has this bug, while later, one of the first betas of 19.20 fixes the issue.

You could go to theme extension manager, and add to CSS extension the following to make your logo appear.

```
.__logo {
	z-index: 9999
}
```

**Comment by rostovtsev** [comment]
> .. to upgrade (which didn't go through well as it broke the UI, but it's ok as it's just a testing server

What exactly was broken? It's very likely that the theme requires few recent changes from Webmin repo to be applied. You can upgrade to the latest Webmin Git version, using update script as stated in the update warning message.
